01-Git_Bash_Make/Task2_bash_script/script

```diff
@@ -1 +1,6 @@
 #!/bin/bash
+
+directory=/tmp/guesanumber
+
+cp -R /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task1_game $directory
```

Comment:
Prefer to use braces syntax for variable expansion in scripts and quotes for whitespace sensitive strings.
like `"${VAR}"`

Reply:
You have hard coded absolute path for the project within repository.
This is definitely looks wrong.
While it works for your current directories, you're commiting it to the repository.
What assumes you expect this repository to be fetched by others to any location where your script won't work due to the hardcoded path.